cmd/crossplane/main.go

```diff
@@ -113,14 +119,22 @@ func main() {
 
 		mgr := setupManager(log, syncPeriod, *extManageTenantKubeconfig)
 
-		log.Info("Adding controllers")
-		kingpin.FatalIfError(stacksControllerSetupWithManager(mgr, *extManageHostControllerNamespace), "Cannot add controllers to manager")
+		if *extManageTemplatesController != "" {
+			*extManageTemplates = true
+		}
+
+		if *extManageTemplates {
+			if *extManageTemplatesController == "" {
+				kingpin.Fatalf("--templates-controller is required with --templates")
```

Comment:
The helm chart will ship a default. The version of that controller will likely need frequent updating so I would rather see it configured in a chart (or other yaml) rather than code.

If we had a good image location that wasn't from a personal repo, I would be more inclined to consider hard coding it (in .go)
.